crates/iota-bridge/src/action_executor.rs

```diff
@@ -1501,25 +1484,27 @@ mod tests {
         }
     }
 
-    #[allow(clippy::type_complexity)]
-    async fn setup() -> (
-        iota_metrics::metered_channel::Sender<BridgeActionExecutionWrapper>,
-        iota_metrics::metered_channel::Sender<CertifiedBridgeActionExecutionWrapper>,
-        IotaMockClient,
-        tokio::sync::broadcast::Receiver<TransactionDigest>,
-        Arc<BridgeOrchestratorTables>,
-        Vec<BridgeAuthorityKeyPair>,
-        IotaKeyPair,
-        BridgeRequestMockHandler,
-        BridgeRequestMockHandler,
-        BridgeRequestMockHandler,
-        BridgeRequestMockHandler,
-        Vec<tokio::task::JoinHandle<()>>,
-        ObjectRef,
-        IotaAddress,
-        Arc<ArcSwap<HashMap<u8, TypeTag>>>,
-        tokio::sync::watch::Sender<IsBridgePaused>,
-    ) {
+    struct Data {
```

Comment:
Just because I hate `Data` on its own, could we call this `SetupData` pls? Just resolve this if you don't want to

Reply:
done